src/Core/Content/Product/SearchKeyword/ProductSearchKeywordAnalyzer.php

```diff
@@ -73,8 +75,12 @@ public function analyze(ProductEntity $product, Context $context, array $configF
      */
     private function tokenize(array $values, Context $context): array
     {
+        $config = $this->configLoader->load($context);
+
+        /** @phpstan-ignore-next-line This ignore should be removed when the deprecated method signature is updated */
```

Comment:
would you mind changing it to this? 
```suggestion
        /** @phpstan-ignore specific.identifier.change.me! (This ignore should be removed when the deprecated method signature is updated) */
```
also see https://phpstan.org/user-guide/ignoring-errors
ignore-next-line will ignore every PHPStan issue and not just the specific one.